starters/svelte-kit-scss/src/routes/(examples)/+layout.svelte

```diff
@@ -1,6 +1,24 @@
-<div>
+<div class="example-container">
   <slot />
   <div>
     <a href="/">Return Home</a>
   </div>
 </div>
+
+<style lang="scss">
+  .example-container {
+    text-align: center;
+
+    div{
```

Comment:
is the div in the context like the footer?
if yes, will it be better to use a class?